### DISCUSSION TITLE
--rate vs. --time-cond

### DISCUSSION THREAD

**Discussion by jidanni** [curl/curl#10724] [Q&A] [answered]
In the curl man page, it isn't clear how --time-cond and
--etag-compare affect --rate .

Does --rate
1. Just limit actual file body transfers? Or,
2. Limit all requests, even if they just
return 304 Not Modified instead of the files themselves?

The man page's --rate section has a mixed terminology,
> transfer starts per time unit (sometimes called request rate)

so one cannot tell.

**Comment by bagder** [comment, marked as answer]
What do you consider "mixed terminology" in the `--rate` man page? I can't see it.

`--rate` limits the request rate. The sending of HTTP requests, which is completely independent of what status codes that is received as a result.

**Reply by jidanni** [reply]
OK, good to know. 
I was just thinking there are  requests, and transfers. So I request something and if my request is honored, then a file will be transferred back, else I'll get a 304... That's the way some people might read the man page.
